tests/console/lvm_thin_check.pm

```diff
@@ -30,6 +31,10 @@ sub run {
 
     $self->select_serial_terminal;
     record_info('INFO', 'Print lvm setup');
+    if (is_tumbleweed) {
```

Comment:
is_tumbleweed is wrong by definitin - Leap and SLE will reach that point when they upgrade to bash 5

Reply:
But we are not there yet

Reply:
> is_tumbleweed is "wrong by definition" (should be avoided at all cost) - Leap and SLE will reach that point when they upgrade to bash 5

That gives us quite some years room to refactor this code 9 times :)

Reply:
From my point of view, we should not hide the problem on Sle and Leap if it will still not be fixed on the time of upgrade. Let's not make our tests green for the sake of making them green. Tests should reflect the state of the system. In case if it will be an issue there also we'll decide additionally how to proceed with it. WDYT?

Reply:
Eve though I do understand the point, we still have `is_tumbleweed` and I personally think it's overkill to avoid it by using `!is_sle && !is_leap` or `is_opensuse && !is_leap` instead. So either we have smarter way to detect issue which is always preferable. But for temporary hot-paches hot to hide other issues, it's overkill to code something sophisticated with enormous costs.